src/log-filestore.c

```diff
@@ -330,33 +332,60 @@ static int LogFilestoreLogger(ThreadVars *tv, void *thread_data, const Packet *p
     if (flags & OUTPUT_FILEDATA_FLAG_OPEN) {
         aft->file_cnt++;
 
-        /* create a .meta file that contains time, src/dst/sp/dp/proto */
-        LogFilestoreLogCreateMetaFile(p, ff, filename, ipver);
+        if (FileWriteMeta()) {
```

Comment:
since a function call is added here it might make sense to actually move that into the LogFilestoreLogCreateMetaFile, etc functions. That keeps the main loop cleaner.